network/hostnetwork/transport.go

```diff
@@ -139,7 +139,7 @@ func (h *hostTransport) SendRequestPacket(ctx context.Context, request network.R
 func (h *hostTransport) RegisterPacketHandler(t types.PacketType, handler network.RequestHandler) {
 	_, exists := h.handlers[t]
 	if exists {
-		panic(fmt.Sprintf("multiple handlers for packet type %s are not supported!", t.String()))
+		log.Warnf("Multiple handlers for packet type %s are not supported! New handler will replace the old one!", t)
```

Comment:
This change doesn't look like something that should be merged to the master branch.